wiki/People/Tournament_Committee/en.md

```diff
@@ -26,7 +33,111 @@ Their responsibilities include:
+
+The Tournament Committee maintains an [account on X](https://twitter.com/osu_tcomm) (formerly Twitter) where the committee broadcasts major changes and updates of various types. Not all changes are communicated here, but many of the most important changes are communicated and clarified through this channel.
+
+The [Community Tournament Status Tracker](https://docs.google.com/spreadsheets/d/1bV5MyrJZI0F52Bx9EcYxdfRu8qGnhWRBKdXHK9uPbO0/edit?usp=sharing) is a publicly viewable spreadsheet detailing the status of all tournaments that have requested official support, i.e. all tournaments that requested screening. This sheet shows all tournaments dating back to January 14th, 2023, and contains a description of the status of the tournament at any given time, including whether a badge was applied or denied. If a badge was denied, the reason for denial is stated on the sheet.
+
+The [Official Tournament Support Updates thread](/community/forums/topics/1715676) is a pinned thread in the [Tournaments & Contests forum](/community/forums/126) which contains all updates to the Official Tournament Support wiki dating back to February 1st, 2023. This thread is updated and bumped every time a change is made to the Official Tournament Support wiki. This thread also contains links to the Official Support wiki and the Community Tournament Status Tracker.
+
+### Adding New Members
+
+New members to the Tournament Committee must meet the following basic criteria before being considered for the position at all:
+- Be at least 18 years old
+- Pass a security check
+
+Whether a prospective member applies via email or is brought forward by an existing member, the process is the same. A thread is either automatically created by the email application or created manually by a member of the Tournament Committee, and the whole committee is notified. The candidate is discussed in-depth, with special consideration paid to the uniqueness of the candidate compared to current members and the candidate’s history in the tournament scene. Assuming discussion is mixed or mostly positive, an anonymous vote is held to determine whether the candidate is added.
+
+### Votes
+
+The Tournament Committee frequently makes use of anonymous voting when making important decisions. Typical uses are listed above, but those are not fully inclusive. The voting requirements also differ based on the use-case. All typical votes also require “strict participation,” meaning that if 75% of the Tournament Committee does not participate by the vote deadline, voting is extended by 24 hours, and the committee is pinged for further participation. Further, all votes require at least an absolute majority, at least 50%, for action to be taken. Votes on many topics have the same requirements, but there are slight differences on some common topics listed below.
```

Comment:
```suggestion
The Tournament Committee frequently makes use of anonymous voting when making important decisions. Typical uses are listed above, but those are not fully inclusive. The voting requirements also differ based on the use case. All typical votes also require “strict participation,” meaning that if 75% of the Tournament Committee does not participate by the vote deadline, voting is extended by 24 hours. Further, all votes require at least an absolute majority, at least 50%, for action to be taken. Votes on many topics have the same requirements, but there are slight differences on some common topics listed below.
```

Reply:
For the record apparently either spelling of use-case/use case is fine, all changes applied.